engine/render/src/dmsdk/render/render.h

```diff
@@ -55,10 +55,14 @@ namespace dmRender
     /*#
      * Font map handle
      * @typedef
-     * @name HFontMap
+     * @name HFont
      */
+    typedef struct FontMap* HFont;
+
+    // Old typedef, used internally. We want to migrate towards HFont
     typedef struct FontMap* HFontMap;
```

Comment:
A quick "rename" (the internal HFontMap is used _a lot_, so I chose to wait with that)